index.js

```diff
@@ -42,7 +42,21 @@ module.exports = class SequelizeTrailpack extends Trailpack {
     _.each(this.models, (model, modelName) => {
       _.each(this.connections, (connection, name) => {
         if (model.connection == name) {
-          this.app.orm[model.globalId] = connection.define(modelName, model.schema, model.config)
+          const Model = connection.define(modelName, model.schema, model.config)
+
+          if (model.config.classMethods) {
```

Comment:
I think config is by default undefined so it will crash here if no config is provided, same for instanceMethods below.